Mathlib/Analysis/NormedSpace/FiniteDimension.lean

```diff
@@ -682,6 +682,12 @@ theorem summable_of_isBigO' {ι E F : Type*} [NormedAddCommGroup E] [CompleteSpa
 set_option linter.uppercaseLean3 false in
 #align summable_of_is_O' summable_of_isBigO'
 
+lemma Asymptotics.IsBigO.comp_summable {ι E F : Type*}
+    [NormedAddCommGroup E] [NormedSpace ℝ E] [FiniteDimensional ℝ E]
+    [NormedAddCommGroup F] [CompleteSpace F]
+    {f : E → F} (hf : f =O[𝓝 0] id) {g : ι → E} (hg : Summable g) : Summable (f ∘ g) :=
+  summable_of_isBigO' hg <| hf.comp_tendsto hg.tendsto_cofinite_zero
```

Comment:
This seems to be useful and not easily derivable from your version:
```
lemma Asymptotics.IsBigO.comp_summable {ι E F} [NormedAddCommGroup E] [NormedAddCommGroup F]
    {f : E → F} (hf : f =O[𝓝 0] id) {g : ι → E} (hg : Summable (‖g ·‖)) : Summable (‖f <| g ·‖) :=
  summable_of_isBigO hg <| hf.norm_norm.comp_tendsto (tendsto_zero_iff_norm_tendsto_zero.mpr hg.tendsto_cofinite_zero)
```
so I think we should probably add it to the other file.

Reply:
Done but now it depends on #8358